mappings/net/minecraft/client/util/Keyboard.mapping

```diff
@@ -2,6 +2,10 @@ CLASS cpj net/minecraft/client/util/Keyboard
 	FIELD a client Lcpk;
 	FIELD b repeatEvents Z
 	FIELD c clipboard Lcns;
+	FIELD d debugCrashStartTime J
+	FIELD e debugCrashLastLogTime J
+	FIELD f debugCrashCounter J
+	FIELD g switchF3State Z
```

Comment:
Is this hardcoded to F3?

Reply:
Not F3, but it's hardcoded to F3+C and only used for debug crashing. Perhaps `switchDebugCrashState`?

Reply:
It's also used to prevent the toggle of the F3 screen when a F3+[something] shortcut is made.